src/plugins/filter/filter-en.hbs

```diff
@@ -781,3 +782,37 @@
 	&lt;li&gt;Alberta&lt;/li&gt;
 	...
 &lt;/ul&gt;</code></pre>
+
+<h3 id="flt-uicontainer">Move UI to specific container</h3>
+
+<p>Set the <code>source</code> parameter to a jQuery selector that you want to hold the filter's UI. The UI will be added to the top (prepend) of said element.</p>
+
+<div class="row">
+	<div class="col-md-6">
+		<ul class="wb-filter" data-wb-filter='{ "source": "#filterUIContainer" }'>
+			<li>Alberta</li>
+			<li>British Columbia</li>
+			<li>Manitoba</li>
+			<li>New Brunswick</li>
+			<li>Newfoundland &amp; Labrador</li>
+			<li>Northwest Territories</li>
+			<li>Nova Scotia</li>
+			<li>Nunavut</li>
+			<li>Ontario</li>
+			<li>Prince Edward Island</li>
+			<li>Quebec</li>
+			<li>Saskatchewan</li>
+			<li>Yukon</li>
+		</ul>
+	</div>
+	<div class="col-md-6">
+		<div id="filterUIContainer"></div>
+	</div>
+</div>
+
+<h4>Source code</h4>
+
+<pre><code>&lt;ul class=&quot;wb-filter&quot; data-wb-filter='{ "source": "#filterUIContainer" }'&gt;
+	&lt;li&gt;Alberta&lt;/li&gt;
+	...
+&lt;/ul&gt;</code></pre>
```

Comment:
It will be helpful to provide a little bit of context on how to use like:

```html
<div class="row">
  <div class="col-md-6">
    <div id="filterUIContainer"></div>
  </div>
  <div class="col-md-6">
     <ul class="wb-filter"....>
       ...
    </ul>
```

Also the filtering interface should be placed before the list, like the data table filtering options.

That bring me another question, to go beyond our minimum requirement, we should apply the property `aria-polite="true"` on the filtered list, like we do for filtering the data table.

Reply:
There is an `aria-live="polite"` on the results count under the text box. I believe this is sufficient to notify the user that the results have been updated.